api/app/scripts/api_unit_test.sh

```diff
@@ -67,33 +67,8 @@ case "$1" in
     php vendor/bin/phpunit -c tests/Unit tests/Unit/Logger/ --coverage-php tests/coverage/logger.cov
     ;;
   selection-all)
-    # Run all three blocks
```

Comment:
This target is only used by the local Makefile, so I changed it to be simpler and more generic, as well as making it produce an HTML report.